source/compatibility.tex

```diff
@@ -60,7 +60,7 @@
 Programs which depend upon \tcode{sizeof('x')} are probably rare.
 
 Subclause \ref{lex.string}:
-\change String literals made const\\
+\change String literals made const.\\
```

Comment:
Below you style `const` as code; should it be code here, too? When do you use which style?